OSDOCS-17319 updates to ESO documentation (#102710)
openshift/openshift-docs: modules/external-secrets-operator-update-channels.adoc

// Module included in the following assemblies:
//
// * security/external_secrets_operator/external-secrets-operator-install.adoc

:_mod-docs-content-type: CONCEPT
[id="external-secrets-operator-update-channels_{context}"]
= Understanding update channels of the {external-secrets-operator}

[role="_abstract"]

> Control the version of the {external-secrets-operator} in your cluster by selecting an update channel. This mechanism allows you to declare a specific version track, helping ensure your environment receives only the updates you require for stability.

Reviewer: ```suggestion
Control the version of the {external-secrets-operator} in your cluster by selecting an update channel. By using this mechanism, you can declare a specific version track, helping ensure your environment receives only the updates you require for stability.
```
